internal/dao/registry.go

```diff
@@ -123,6 +124,77 @@ func AccessorFor(f Factory, gvr client.GVR) (Accessor, error) {
 	return r, nil
 }
 
+func GVRForKind(apiVersion string, kind string) (string, bool, error) {
+	type gvrInfo struct {
+		gvr        string
+		namespaced bool
+	}
+
+	gvrs := map[string]gvrInfo{}
+
+	switch v := apiVersion; v {
+	case "apps/v1":
+		gvrs = map[string]gvrInfo{
+			"ReplicaSet":  {"apps/v1/replicasets", true},
+			"DaemonSet":   {"apps/v1/daemonsets", true},
+			"StatefulSet": {"apps/v1/statefulsets", true},
+			"Deployment":  {"apps/v1/deployments", true},
+			"Jobs":        {"apps/v1/jobs", true},
+			"CronJobs":    {"apps/v1/cronjobs", true},
+		}
+	case "v1":
```

Comment:
I don't think a node/replicaset can be a controllers?
Best to keep a static map as pointed above and infer K-R from there?

Reply:
> I don't think a node/replicaset can be a controllers?

I was a bit surprised myself, but I found this in my default Docker Desktop installation with Kubernetes enabled.

```
apiVersion: v1
kind: Pod
metadata:
  name: etcd-docker-desktop
  namespace: kube-system
  ownerReferences:
  - apiVersion: v1
    controller: true
    kind: Node
    name: docker-desktop
```

Reply:
@gitolicious I stand corrected! Right static pods I think the controller would be the node.